app/experimental_appconfig.go

```diff
@@ -0,0 +1,57 @@
+// +build !stable
+
+package app
+
+import (
+	"github.com/cosmos/cosmos-sdk/types/module"
+
+	"github.com/regen-network/regen-ledger/x/data"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+func setCustomModuleBasics() []module.AppModuleBasic {
+	return []module.AppModuleBasic{
+		data.AppModuleBasic{},
+		ecocredit.AppModuleBasic{},
+		group.AppModuleBasic{},
+	}
```

Comment:
this is an experimental build, so we integrate new modules here

Reply:
you will probably need another function to account for the new module wiring and `setCustomKeepers` might be left empty for regen modules since they don't actually have a keeper.